docs/docs/concepts/Cluster_Support.md

```diff
@@ -0,0 +1,108 @@
+---
+title: "Cluster Support"
+linkTitle: "Cluster Support"
+weight: 4
+description: >
+    Cluster support for Triggers and Functiosn
+---
+
+**Notice** on oss cluster, before executing any gears function, you must send `REDISGEARS_2.REFRESHCLUSTER` command to all the shards so that all the shards will be aware of the cluster topology. Without this step, each shard will act as a single oss instance.
```

Comment:
```suggestion
**Notice**: On an OSS cluster, before executing any gears function, you must send a `REDISGEARS_2.REFRESHCLUSTER` command to all the shards so that they will be aware of the cluster topology. Without this step, each shard will act as a single OSS instance.
```

Reply:
Not sure about "oss" vs "OSS". Let me know what you prefer here.